app/Orchid/Layouts/Batch/AddFarmersLayout.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Orchid\Layouts\Batch;
+
+use App\Models\Batch\Farmers;
+use App\Models\User;
+use Orchid\Screen\Field;
+use Orchid\Screen\Layouts\Rows;
+use Orchid\Screen\Fields\Relation;
+
+class AddFarmersLayout extends Rows
```

Comment:
Should be called `BatchEditFarmersLayout`

Reply:
Renamed 'AddFarmersLayout' to 'BatchEditFarmersLayout'. Also renamed it on the BatchEditScreen.